src/Test/Perf/util/DownloadCPC_util.csx

```diff
@@ -0,0 +1,29 @@
+#load "test_util.csx"
+
+using System;
+using System.IO;
+
+public void DownloadCPC()
+{
+    var cpcDestinationPath = GetCPCDirectoryPath();
+    var cpcSourceBinaryLocation = GetCPCSourceBinaryLocation();
+    
+    
+    // Delete the existing CPC folder
+    if (Directory.Exists(cpcDestinationPath))
+    {
+        Directory.Delete(cpcDestinationPath, true);
+    }
+    
+    // Copy CPC from the share to cpcDestinationPath
+    CopyDirectory(cpcSourceBinaryLocation, cpcDestinationPath);
+}
+
+void CopyDirectory(string source, string destination)
+{
+    var result = ShellOutUsingShellExecute("Robocopy", $"/mir {source} {destination}");
+    if (!result.Succeeded)
```

Comment:
I don't think this is right.  Robocopy will return non-zero exit codes for a lot of cases where we don't want to throw (see http://ss64.com/nt/robocopy-exit.html).  That said, per other comment, I think this probably belongs in bootstrap.bat, and we can just remove this.